python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_decorator.py

```diff
@@ -34,6 +35,45 @@ def all_dbt_assets(context, dbt: DbtCli):
     ).success
 
 
+def test_json_manifest():
```

Comment:
Can we just combine make this into one test, and just parameterize `manifest`?

```suggestion
@pytest.mark.parametrize("manifest", ...)
def test_dbt_assets_from_manifest():
```